### DISCUSSION TITLE
Delete item not getting called on our back end

### DISCUSSION THREAD

**Discussion by box-turtle** [microsoft/Microsoft-Fabric-workload-development-sample#154] [Help]
When I delete an item in the workspace view, our back end implementation of [the delete API](https://learn.microsoft.com/en-us/rest/api/fabric/workload/workloadapi/item-lifecycle/delete-item?tabs=HTTP) is not getting called.

I believe this was working before, not sure when the behavior changed.

![image](https://github.com/user-attachments/assets/7297aa88-2f69-442e-8fb5-1e1fc6ebe135)

Is there some configuration or action we need to implement to have this work? I thought maybe the new tab onDelete callback but that doesn't seem to get called either?

**Comment by tjroberts** [comment]
Any updates on this?

**Comment by shaym83** [comment]
Discussed over email; the issue is probably that in DELETE API, Fabric sends an S2S token, and not the user token.
If the workload attempts to validate that token as a user token, it will cause the issue described in this discussion.
